open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/graph/LineageGraphConnector.java

```diff
@@ -637,8 +676,7 @@ public LineageResponse lineage(Scope scope, String guid, String displayNameMustC
                 lineageVerticesAndEdges = helper.sourceAndDestination(guid, includeProcesses);
                 break;
             case END_TO_END:
-                lineageVerticesAndEdges = helper.endToEnd(guid, includeProcesses, EDGE_LABEL_SEMANTIC_ASSIGNMENT,
-                        EDGE_LABEL_DATAFLOW_WITH_PROCESS, EDGE_LABEL_CLASSIFICATION);
+                lineageVerticesAndEdges = helper.endToEnd(guid, includeProcesses, EDGE_LABEL_DATAFLOW_WITH_PROCESS);
```

Comment:
For consistency, better to remove the ... parameter from endToEnd method and have it within.